### PR TITLE
MM-8681 Adding client ability to use separate websocket endpoint.

### PR DESCRIPTION
**Pull request by crspeller**
Client compatibility changes for https://github.com/mattermost/mattermost-server/pull/8307

### CODE REVIEW

File: actions/websocket_actions.jsx [hunk 1]
```diff
@@ -45,22 +45,27 @@ export function initialize() {
         return;
     }
 
-    let connUrl = getSiteURL();
+    let connUrl = '';
+    if (global.window.mm_config.WebsocketURL === '') {
```
Comment (enahum) [inline actions/websocket_actions.jsx]: Can we remove the usage of the global variable here?

Comment (crspeller) [reply actions/websocket_actions.jsx]: Ya. Actually lets just wait for Jesse's PRs to be merged, then I can rebase ontop of that.

Comment (jasonblais) [reply actions/websocket_actions.jsx]: @crspeller Those PRs are merged now

Comment (lieut-data) [reply actions/websocket_actions.jsx]: The `actions/` subdirectory is part of Phase 2, which I'm tackling presently. If it's easier for everyone, I don't mind this change going in as-is and I can clean this file up holistically.

Comment (crspeller) [reply actions/websocket_actions.jsx]: Ok. If that's easier for you let's do that.

**Pull request merged**